configs/stac/stac_fly_tethered.yaml

```diff
@@ -0,0 +1,13 @@
+fit_path: "Fruitfly_fit.p"
+transform_path: "transform_mocap_fly.h5"
+data_path: "tests/data/bout_dict.h5"
```

Comment:
@elliottabe what is this file bout_dict.h5?

Reply:
This and the csv are the data files that have the keypoints. 
Treadmill:
https://datadryad.org/stash/dataset/doi:10.5061/dryad.mpg4f4r73
Tethered:
https://datadryad.org/stash/dataset/doi:10.5061/dryad.nzs7h44s4

Reply:
@jf514 I got the treadmill csv below, thanks! But I couldn't find the bout_dict.h5. Can you please write some explicit instructions here for how to obtain data that will work here? If that's too convoluted, can you create a truncated file we can check in? Needs to be less than 50 Mb.

Reply:
The bout_dict one is from unpublished data but the data from dryad is equivalent from another paper. It can be pretty much the same way as the csv by just grabbing the key point data.  Here is essentially how to get the data: 
``` 
full_df = pd.read_parquet(data_path, engine='pyarrow')
bout_stats = full_df.groupby(['walking_bout_number','fullfile','Sex'])[['fictrac_delta_rot_lab_y_mms', 'fictrac_delta_rot_lab_z_deg/s']].agg(['mean','min','max','std','count'])
fast = (bout_stats[('fictrac_delta_rot_lab_y_mms','mean')] >= 12) & (bout_stats[('fictrac_delta_rot_lab_y_mms','min')] >= 10)
straight = (bout_stats[('fictrac_delta_rot_lab_z_deg/s','mean')].abs() <= 45) &\
           (bout_stats[('fictrac_delta_rot_lab_z_deg/s','min')] >= -60) &\
           (bout_stats[('fictrac_delta_rot_lab_z_deg/s','max')] <= 60)

legs = ['T1_left', 'T1_right', 'T2_left', 'T2_right', 'T3_left', 'T3_right']
joints = ['coxa', 'femur', 'tibia', 'tarsus']
xpos_geoms = ['coxa', 'femur', 'tibia', 'tarsus', 'claw']
joint_names = [f'{joint}_{leg}' for leg in legs for joint in joints]
xpos_names = [f'{joint}_{leg}' for leg in legs for joint in xpos_geoms]
# physics.named.data.framepos[pos_names]
site_names = [f'tracking[{joint_name}]' for joint_name in xpos_names]

legs_data = ['L1', 'R1', 'L2','R2', 'L3','R3']
joints_data = ['A','B','C','D','E']
coords_data = ['_x','_y','_z']
joint_pos_columns = [leg + joint + coord 
                     for leg in legs_data
                     for joint in joints_data 
                     for coord in coords_data]

straight_bouts = bout_stats[fast & straight].index
straight_bout_num = np.array([i[0] for i in straight_bouts],dtype=int)
mean_straight_bout = (full_df[full_df['walking_bout_number'] == 14574][joint_pos_columns].values.reshape(-1,30, 3)).mean(axis=0)
ref_n = 14574 # Some example bout number
ref_bout = (full_df[full_df['walking_bout_number'] == ref_n][joint_pos_columns].values.reshape(-1,30, 3))
bout_dict = {'walking_bout{:02}'.format(n):{} for n in range(len(straight_bout_num))}

for n, bout_num in enumerate(straight_bout_num):
    bout = full_df[full_df['walking_bout_number'] == bout_num]
    bout_dict['walking_bout{:02}'.format(n)]['orig_xpos'] = bout[joint_pos_columns].values.reshape(-1,30, 3)
    bout_dict['walking_bout{:02}'.format(n)]['orig_xpos'] = bout_dict['walking_bout{:02}'.format(n)]['orig_xpos'] + (mean_straight_bout-np.mean(bout_dict['walking_bout{:02}'.format(n)]['orig_xpos'],axis=0))
```

Reply:
Ok. Prepping that data is beyond the scope of this commit but we'll need a working example for the next PR.

Reply:
Btw, I am reading between the lines here to understand that "unpublishded data" means you can't share it?

Reply:
Um ya I need to ask permission first before sharing data